opendbc/generator/honda/honda_accord_s2t_2018_can.dbc

```diff
@@ -11,6 +11,12 @@ BO_ 446 BRAKE_MODULE: 3 VSA
  SG_ COUNTER : 21|2@0+ (1,0) [0|3] "" XXX
  SG_ CHECKSUM : 19|4@0+ (1,0) [0|15] "" XXX
 
+BO_ 662 SCM_BUTTONS: 8 SCM
```

Comment:
@energee , this shouldn't be here as it's already in _bosch_2018.dbc.
Wondering why in the generated file 662 isn't repeated twice.

Reply:
Also, the VAL line should be moved from the individual dbc file portions to here.

Reply:
Yea, that should be correct in opendbc, at one point i thought this had a different packet size on accord so i pulled  it out of the bosch dbc, but I must have been looking quickly at another bus, it matches the other cars. I will remerge the subtree. Did you want me adjust the VAL lines on OpenDBC?